templates/ai/mlops.html

```diff
@@ -0,0 +1,210 @@
+{% extends "ai/_base_ai.html" %}
```

Comment:
```suggestion
{% extends "ai/base_ai.html" %}
```
Our convention is to name these without a leading underscore(_)